### PR TITLE
Added Kvantum GUI settings

### PR DESCRIPTION
**Pull request by Teeffelen**
Features added:

* Kvantum toggle box
* Kvantum light-theme input field
* Kvantum dark-theme input field

Bugs fixed:

* The `install.sh` tried to write a file to `/home/$SUDO_USER/.local/share/applications/` which did not exist yet (on my PC), added a check to create the directory if it was not found before writing the `.desktop` file.

Tested on:

* Plasma - 5.19.3
* Qt - 5.15.0
* ArchLinux - 5.7.7

> **NOTE:** I did mess up the logo in the `mainwindow.ui` (QT-Designer could not find it under `/opt/yin-yang/src/ui/assets/yin-yang.svg`), it is displayed but not like the original (needs fixing before release).

Edit: added screenshot:
image

### CODE REVIEW

File: src/ui/mainwindow.py [hunk 1]
```diff
@@ -2,34 +2,30 @@
...
-            MainWindow.sizePolicy().hasHeightForWidth())
+        sizePolicy.setHeightForWidth(MainWindow.sizePolicy().hasHeightForWidth())
         MainWindow.setSizePolicy(sizePolicy)
         MainWindow.setMinimumSize(QtCore.QSize(300, 360))
         MainWindow.setMaximumSize(QtCore.QSize(300, 360))
         MainWindow.setBaseSize(QtCore.QSize(300, 360))
         icon = QtGui.QIcon()
-        icon.addPixmap(QtGui.QPixmap("/opt/yin-yang/src/ui/assets/yin-yang.svg"),
-                       QtGui.QIcon.Normal, QtGui.QIcon.Off)
+        icon.addPixmap(QtGui.QPixmap("../../../../../../opt/yin-yang/src/ui/assets/yin-yang.svg"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
```
Comment (Teeffelen) [inline src/ui/mainwindow.py:28]: This needs to be fixed before a release.

File: src/ui/mainwindow.py [hunk 2]
```diff
@@ -41,28 +37,25 @@ def setupUi(self, MainWindow):
...
         self.yinyang_img.setSizeIncrement(QtCore.QSize(0, 0))
         self.yinyang_img.setBaseSize(QtCore.QSize(0, 0))
         font = QtGui.QFont()
         font.setPointSize(20)
         self.yinyang_img.setFont(font)
         self.yinyang_img.setText("")
         self.yinyang_img.setTextFormat(QtCore.Qt.RichText)
-        self.yinyang_img.setPixmap(QtGui.QPixmap("/opt/yin-yang/src/ui/assets/icon.png"))
-        self.yinyang_img.setScaledContents(True)
+        self.yinyang_img.setPixmap(QtGui.QPixmap("assets/icon.png"))
```
Comment (Teeffelen) [inline src/ui/mainwindow.py:54]: This also needs to be fixed (probably)

**Comment by Teeffelen** [comment]
The path issues I have are probably because I opened the `.ui` files from the GIT directory, not from the `/opt/yin-yang/` directory -- my bad.

**Comment by daehruoydeef** [comment]
Great work! I currently have another branch running called 1.5 it has new features such as more GUI options and react better to desktop specific behavior. I also added Kvantum support but did not got around to test it yet fully. Thanks

**Comment by daehruoydeef** [comment]
`gsettings` will only be used on the gtk platform and not on kde. I came up with my own solution on kde basically changing the theme prop inside the `gtk-3.0.ini `file.
If you want you can make your changes to the `kdegtk.py` file inside the `plugin` folder and see if this is working. At the time of creating this I was not aware of another way to change gtk theme in KDE.

https://github.com/daehruoydeef/Yin-Yang/blob/master/src/plugins/gtkkde.py

Your solution should be really easy to implement. Would be really cool if you could test if this is live reloading the gtk applications also

**Pull request merged**